src/pages/trade/ui/chart.tsx

```diff
@@ -260,7 +261,13 @@ export const Chart = observer(() => {
         ))}
       </div>
 
-      {error && <div className='text-white'>Error loading pair selector: ${String(error)}</div>}
+      {error && (
+        <div className='w-full h-full flex items-center justify-center'>
+          <div className='w-[450px] h-full flex items-center justify-center'>
+            <BlockchainError onDetailsClick={() => console.log('Details clicked')} />
```

Comment:
question: in the near future, this should open the dialog, right?

Reply:
Yes, that's the next step for this PR, I wanted a preliminary review regardless.